README.md

```diff
@@ -120,6 +120,14 @@ Private keys can be encrypted using one of the following cipher methods:
 * hmac-sha2-256-etm<span></span>@openssh.com
 * hmac-sha2-512-etm<span></span>@openssh.com
 
+
+## Compression
+
+**SSH.NET** supports the following compression algorithms:
+* none (default)
+* zlib<span></span>@openssh.com (.NET 6 and higher)
```

Comment:
Is there another way to escape the URL? It doesn't work in the nuget README: https://www.nuget.org/packages/SSH.NET/#readme-body-tab

Reply:
Let me try `[zlib@openssh.com]()` -> [zlib@openssh.com]()

Reply:
try `zlib&#64;openssh.com` -> zlib&#64;openssh.com

Reply:
try `zlib<nolink>@openssh.com` -> zlib<nolink>@openssh.com

Reply:
> try `zlib<nolink>@openssh.com` -> zlib@openssh.com

probably not work in nuget...